### ISSUE TITLE
minus to learning rate

### ISSUE THREAD

**Issue by sumorday** [arogozhnikov/DeepMMD-GAN#2]
Heyl, if I used the negative learning rate then there was a bug. invalid learning rate. what happened. 

# Yup, just added minus to learning rate - and that's where you get adversarial training :)
optimizerD = optim.Adam(netD.parameters(), lr= -1e-4, betas=(0.5, 0.999))

**In discriminator we need do gradient descent,right? but error happened.**

**Comment by arogozhnikov** [comment]
Likely new versions of pytorch forbid using negative learning rate, that's
the problem

**Comment by sumorday** [comment]
Lmao
So sad. then how can I do....
